### PR TITLE
Updating to CLDR 39.0.0

### PR DESCRIPTION
**Pull request by sffc**

### CODE REVIEW

File: resources/testdata/Cargo.toml [hunk 1]
```diff
@@ -114,7 +114,7 @@ cldr_json_glob = [
 ]
 
 # Git tag or sha1 for the CLDR data used to generate the testdata.
-gitref = "39.0.0-BETA2"
+gitref = "39.0.0-BETA3"
```
Comment (srl295) [inline resources/testdata/Cargo.toml:117]: Obsolete, that is _so_ two hours ago. 😄

File: resources/testdata/data/cldr/cldr-dates-full/main/en-CA/timeZoneNames.json [hunk 2]
```diff
@@ -1144,9 +1144,6 @@
               "Broken_Hill": {
                 "exemplarCity": "Broken Hill"
               },
-              "Currie": {
-                "exemplarCity": "Currie"
-              },
```
Comment (srl295) [inline resources/testdata/data/cldr/cldr-dates-full/main/en-CA/timeZoneNames.json:1149]: stale.

**Review by srl295 — DISMISSED** [review]
i approve, but 39.0.0 is even better

File: resources/testdata/data/cldr/cldr-dates-full/main/en-CA/timeZoneNames.json (hunk 2 above)
Comment (sffc) [reply resources/testdata/data/cldr/cldr-dates-full/main/en-CA/timeZoneNames.json:1149]: I think this is a change from 38 to 39 on a file that was never updated to 39.0.0-BETA2 within the ICU4X repo.

**Review by srl295 — APPROVED** [review]

**Pull request merged**